pcdsdevices/mpod_apalis.py

```diff
@@ -64,7 +73,7 @@ def set_voltage(self, voltage_number):
         """
         max_voltage = self.max_voltage.get()
 
-        if voltage_number <= max_voltage:
+        if abs(voltage_number) <= abs(max_voltage):
             self.voltage.put(voltage_number)
```

Comment:
This code isn't from you, but I am very confused here. Based on the PV, the `voltage` is a measured value. Shouldn't this be putting a value to `voltage_setpoint` ... which is `EpicsSignalRO` for some reason?

Reply:
Re-reading the PR description makes the last part more clear, but the PV naming in this IOC seems very odd.